shell/browser/ui/x/x_window_utils.cc

```diff
@@ -4,6 +4,9 @@
 
 #include "shell/browser/ui/x/x_window_utils.h"
 
+#include <X11/Xlib.h>
```

Comment:
What requires this header here ? It contradicts with the refactor from upstream, whatever dependency on Xlib headers are needed should now come from `//ui/gfx/x` , the xproto* headers have macros that will conflict with Xlib headers as upstream starts to remove the undefs

Reply:
@deepak1556 it was consistently failing on https://github.com/electron/electron/blob/8ab61d3cdeff2840ad1144ce9b8ac33227a6dad7/shell/browser/ui/x/x_window_utils.cc#L89

saying it was not recognized

Reply:
think i got it!